internal/impl/confluent/processor_schema_registry_decode_test.go

```diff
@@ -316,10 +316,10 @@ func TestSchemaRegistryDecodeAvro(t *testing.T) {
 
 	cfg := decodingConfig{}
 	cfg.avro.rawUnions = false
-	goAvroDecoder, err := newSchemaRegistryDecoder(urlStr, noopReqSign, nil, cfg, service.MockResources())
+	goAvroDecoder, err := newSchemaRegistryDecoder(urlStr, noopReqSign, nil, cfg, 10*time.Minute, service.MockResources())
```

Comment:
Maybe add a constant for the 10m

Reply:
Yeah I removed then put it back, will fix all the hardcoding